kuryr_kubernetes/controller/drivers/namespace_security_groups.py

```diff
@@ -34,13 +34,14 @@
                       "namespaces into the default namespace.")),
     cfg.StrOpt('sg_allow_from_default',
                help=_("Default security group to allow traffic from the "
-                      "default namespaces into the other namespaces."))
+                      "default namespaces into the other namespaces.")),
+    cfg.ListOpt('global_namespace',
```

Comment:
This should probably by `global_namespaces`.

Reply:
+1